app/controllers/companies_controller.rb

```diff
@@ -27,13 +28,11 @@ def welcome
   private
 
   def company_params
-    temp = params.require(:company).permit(:name,
-                                    :organization,
-                                    :title,
-                                    :state,
-                                    :city,
-                                    :email,
-                                    :hiring,
-                                    :hire_count )
+    cp = params.require(:company).permit(:name,:organization,:title,:state,:city,:email)
```

Comment:
previously we were taking in the "hire_count" parameter even if the user didn't fill in anything. In that case it comes in as an empty string, so it would fail the validation on numericality for that field. this way is not pretty but it is closer to what I think is the intended logic, which is that you only fill in `hiring` and `hire_count` if they check it